pvlib/modelchain.py

```diff
@@ -589,8 +665,9 @@ def dc_model(self, model):
             model = model.lower()
             if model in _DC_MODEL_PARAMS.keys():
                 # validate module parameters
-                missing_params = (_DC_MODEL_PARAMS[model]
-                                  - set(self.system.module_parameters.keys()))
+                missing_params = (
+                    _DC_MODEL_PARAMS[model] -
+                    _common_keys(self.system.module_parameters))
```

Comment:
Thanks, `_common_keys` is much more clear to me. Can we now remove `_check_consistent_params`? I don't see what it's protecting against.

Also maybe change the error message below to "...one or more required parameters are missing from one or more Arrays: "

Reply:
I think we can remove it, unless we want one consistent place where users can get an error for this reason.

Reply:
Removed.